web/src/locales/de/data.json

```diff
@@ -108,6 +108,8 @@
   "cert": {
     "Bit size": "Bitgröße",
     "Bit size - Tooltip": "Länge des Secret-Keys",
+    "CA Certificate": "CA Certificate",
```

Comment:
I don't like the `CA Certificate` name, all certicates are coming from a "CA", it doesn't show any distinct values. Even `LDAP` is a better name

Reply:
I don`t like LDAP, because in the future this scope can be used in other ssl connections. For example OpenIdConnect by SSL when casdor deployed on docker. Maybe a Trusted Root Certificate sounds better?